packages/layout/src/node/shouldBreak.js

```diff
@@ -1,14 +1,23 @@
 /* eslint-disable no-continue */
+/* eslint-disable no-restricted-syntax */
 
 import * as R from 'ramda';
+import * as P from '@paladin-analytics/rpdf-primitives';
 
 import getWrap from './getWrap';
 import getNodesHeight from './getNodesHeight';
+import splitText from '../text/splitText';
+// eslint-disable-next-line import/no-cycle
```

Comment:
this warns about the import dependency cycle between this file and `resolvePagination.js` file, but we can ignore this as this won't affect the JS parsing

Reply:
Do you think that it matches well if we move the `splitView` (and any associated functions in `resolvePagination.js`) to `/layout/view/splitView`?